client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -29,6 +33,23 @@ interface Props {
 export default function Login({ handleSubmit }: Props): JSX.Element {
   const classes = useStyles();
 
+  const { updateLoginContext } = useAuth();
```

Comment:
It would be better if you can componentize the demo user button along with this function? As we are using the same code in multiple places :)